### ISSUE TITLE
Script runs without errors, but has no effect

### ISSUE THREAD

**Issue by PhoenixSmaug** [paul-ridgway/aero-keyboard#4]
All scripts I run are executed without errors, for example:
```
$ ./white.rb
Opening device 1:5:3...
Closing device
```

But it has absolutely no effect on the keyboard backlight, even if I run it with sudo or perform the complete setup process as root user. I have a AERO 15 YB and I noticed that my keyboard is device 005 and not 004 (`Bus 001 Device 005: ID 1044:7a3b Chu Yuen Enterprise Co., Ltd USB-HID Keyboard`), but this should not really matter for the code.

**Comment by paul-ridgway** [comment]
Have you got the backlight enabled using fn + space, from memory, I don't think it turns them on automatically just sets the pattern. If that doesn't work you'd have to debug like I did eg using a Windows VM and WireShark

**Comment by PhoenixSmaug** [comment]
Yes, the backlight is enabled. It is still the profile I set before installing Linux on the machine. I'm currently working on setting up the Control Center in a VM, I installed it but it can't be open (common problem, I'm currently trying different fixes from forum posts).

I also noticed that unlike in your example, when I run the script I get no HIDAPI output, only the information of the opening and closing of the device

**Comment by paul-ridgway** [comment]
I think your vendor/product ID might differ from mine - try update this [line](https://github.com/paul-ridgway/aero-keyboard/blob/767ebd0f91149e137122349828433f306a230f1e/keyboard/device.rb#L80)

**Comment by PhoenixSmaug** [comment]
I'm pretty certain the IDs are correct, when I change them I don't get the `Opening device 1:5:3...` anymore.

Since I didn't got the Control Center working in the VM, I installed Windows on the second drive to boot it regularly. Now the Control Center installation worked perfectly, but the keyboard lighting also does not respond to changes done in the program. So I assume their is an external problem - maybe even a hardware error, that prevents the keyboard from changing its lighting settings.

I've decided that it's probably not worth the effort to track down this deeper problem or send the laptop with warranty to Gigabyte.
